### PR TITLE
Align line lengths and quotes in documentation

### PR DESCRIPTION
**Pull request by tyrann0us**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update

**What is the current behavior?** (You can also link to an open issue here)
* Markdown files have varying line lengths
* Quotes in tables use double and single quotes inconsistently
* Some tables have invalid Markdown table syntax

**What is the new behavior (if this is a feature change)?**
* Markdown files use 100-character line length consistently
* Quotes are aligned to single quotes
* Correct Markdown table syntax for all tables

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

**Other information**:

### CODE REVIEW

**Review by Chrico — APPROVED** [review]

File: docs/wp-scripts.md [hunk 1]
```diff
@@ -3,15 +3,16 @@
 ## Lint
 
 This workflow runs [ESLint](https://eslint.org/), [Stylelint](https://stylelint.io/),
-and [markdownlint](https://github.com/DavidAnson/markdownlint) wrapped in the
-the [`@wordpress/scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/)
+and [markdownlint](https://github.com/DavidAnson/markdownlint) wrapped in the the [
+`@wordpress/scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/)
```
Comment (meszarosrob) [inline docs/wp-scripts.md:7]:
```suggestion
and [markdownlint](https://github.com/DavidAnson/markdownlint) wrapped in the
[`@wordpress/scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/)
```

**Review by meszarosrob — APPROVED** [review]

File: docs/wp-scripts.md (hunk 1 above)
Comment (tyrann0us) [reply docs/wp-scripts.md:7]: Thanks for the suggestion, which I agree. However, PhpStorm automatically applied all changes, and I'd rather stick to that if that's okay. There are other occurrences of weird line separations, like here: https://github.com/inpsyde/reusable-workflows/pull/156/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R9-R10 and here: https://github.com/inpsyde/reusable-workflows/pull/156/files#diff-c55cc7d1f7f61335e49de6bc171223be1afa8985e9d9a81b536081bd9a65fd8dR34-R35.

Comment (meszarosrob) [reply docs/wp-scripts.md:7]: Sure. Then maybe we can just remove the 2x "the".

Comment (tyrann0us) [reply docs/wp-scripts.md]: Sorry, I totally overlooked that. 🤦🏽‍♂️

**Pull request merged**